testing/python/test_closures.py

```diff
@@ -0,0 +1,145 @@
+"""WIP: this test class will be moved to fixture.py in this module"""
```

Comment:
Temporarily using a separate module for those tests because it makes switching between `master` and this branch to run tests.